### PR TITLE
Punish architect for tasks boosting #220

### PR DESCRIPTION
**Pull request by neonailol**

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#585](https://github.com/zerocracy/farm/issues/585) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#585](https://github.com/zerocracy/farm/issues/585) is assigned to @g4s8[/z](https://www.0crat.com/u/g4s8), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/481070294). The budget is 15 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§27](http://datum.zerocracy.com/pages/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

**Comment by neonailol** [comment]
@g4s8 update my pr, can't verify is test ok or not, as `mvn test` fails with some obscure messages on master branch of this project

**Comment by neonailol** [comment]
@g4s8 i found source of problem, i do not have `mongod` in path, i think for tests there should be an embedded version of mongo running, should i submit an issue?

**Comment by g4s8** [comment]
>  i think for tests there should be an embedded version of mongo running

@neonailol I think it can make sense

**Comment by g4s8** [comment]
@yegor256 I accept

**Comment by yegor256** [comment]
@rultor merge

**Comment by neonailol** [comment]
@yegor256 please look at this

File: src/main/resources/com/zerocracy/stk/pm/cost/penalize_for_boosting.groovy [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import com.zerocracy.pm.ClaimOut
+import com.zerocracy.pm.staff.Roles
+
+def exec(Project project, XML xml) {
+  new Assume(project, xml).notPmo()
+  new Assume(project, xml).type('Set boost')
+  ClaimIn claim = new ClaimIn(xml)
+  String job = claim.param('job')
+  if (claim.hasAuthor() && new Roles(project).allRoles(claim.author()).contains('ARC')) {
+    new ClaimOut()
```
Comment (yegor256) [inline src/main/resources/com/zerocracy/stk/pm/cost/penalize_for_boosting.groovy]: @neonailol use `claim.copy()` instead, see how it's done in all other groovy scripts: 7d9e1d9503d3a432054549cd053728cecb35efe4

File: src/main/resources/com/zerocracy/stk/pm/cost/penalize_for_boosting.groovy [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      .type('Make payment')
+      .param('cause', claim.cid())
```
Comment (yegor256) [inline src/main/resources/com/zerocracy/stk/pm/cost/penalize_for_boosting.groovy]: @neonailol and remove this

**Comment by yegor256** [comment]
@neonailol see above

**Comment by neonailol** [comment]
@yegor256 correct code, as you suggested, it's kinda different now from other penalize scripts, 
examples:
https://github.com/zerocracy/farm/blob/master/src/main/resources/com/zerocracy/stk/pm/cost/penalize_for_manual_assignment.groovy
https://github.com/zerocracy/farm/blob/master/src/main/resources/com/zerocracy/stk/pm/cost/penalize_for_refusal.groovy
https://github.com/zerocracy/farm/blob/master/src/main/resources/com/zerocracy/stk/pm/cost/penalize_for_same_user_github_assignment.groovy

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) this job was assigned to you 5 days ago. It will be taken away from you soon, unless you close it, see [§8](http://datum.zerocracy.com/pages/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by neonailol** [comment]
@yegor256 fixed the test, do not why it failed on rultor, and not fail on travis

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) this job was assigned to you 8 days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by g4s8** [comment]
@yegor256 please close this PR since a performer was resigned from the job #220

**Comment by yegor256** [comment]
@rultor merge

**Comment by g4s8** [comment]
@yegor256 this PR failed to merge along of broken tests, maybe it's time to close it?

**Comment by yegor256** [comment]
@neonailol @g4s8 closing

**Comment by 0crat** [comment]
Order was finished, quality was "acceptable": +15 points just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8), total is [+4615](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
The job [#585](https://github.com/zerocracy/farm/issues/585) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256), total is [+10854](https://www.0crat.com/u/yegor256)

**Pull request closed without merging**